### PR TITLE
Adding Tests to the TestPlan

### PR DESCRIPTION
**Pull request by alajko**
TODO
- [ ] Add Attack/Security Test Cases

### CODE REVIEW

File: specs/test/testplan.md [hunk 1]
```diff
@@ -109,6 +102,26 @@ It should cover the basic function to pass, to fail, and error conditions.
 * note
 * automated?
 ---
+
+### smartcontract
+
+* test case # : SC1
+* description : deploy ERC20 smart contract in 1 shard
+* test procedure : smart contract is deployed in a automated fashion e.g. faucet contract
```
Comment (LeoHChen) [inline specs/test/testplan.md]: faucet contract is not ERC20, right?
we need to create an ERC20 contract

File: specs/test/testplan.md [hunk 2]
```diff
@@ -168,16 +199,55 @@ It should cover the basic function to pass, to fail, and error conditions.
...
-* passing criteria
+* test procedure : increase number of txns in block from 1000 to 10,000, stepwise
+* evaluation: change in block latency per shard/change in transactions per sec, per shard.
+* dependency
+* note
+* automated?
+---
+* test case # : STP2
+* description : increasing number of nodes in a shard
+* test procedure : increase number of nodes in a shard from 100 to 5000
```
Comment (LeoHChen) [inline specs/test/testplan.md]: don't think we can have one shard with 5000 nodes. Maybe up to 1000?

File: specs/test/testplan.md [hunk 3]
```diff
@@ -168,16 +199,55 @@ It should cover the basic function to pass, to fail, and error conditions.
...
+* dependency
+* note
+* automated?
+---
 
 ### networking level stress
+
+* test case # : NT1
+* description : 
+* test procedure : start consensus with a peer-p2p network of 50 peers, increase it to 1000 peers, send 1 ping message every 10 secs.
```
Comment (LeoHChen) [inline specs/test/testplan.md]: for stress, we probably want to stress more, like 1 ping message every 100 ms?

File: specs/test/testplan.md (hunk 2 above)
Comment (alajko) [reply specs/test/testplan.md]: Done.

File: specs/test/testplan.md (hunk 3 above)
Comment (alajko) [reply specs/test/testplan.md]: Done

File: specs/test/testplan.md (hunk 1 above)
Comment (alajko) [reply specs/test/testplan.md]: Fixed.

**Pull request merged**